scenarios/openshift/etcd.yml

```diff
@@ -3,8 +3,4 @@
   config:
     namespace_pattern: ^openshift-etcd$
     label_selector: k8s-app=etcd
-- id: wait-for-pods
-  config:
-    namespace_pattern: ^openshift-etcd$
-    label_selector: k8s-app=etcd
-    count: 3
+    krkn_pod_recovery_time: 30
```

Comment:
Tullio, are we able to increase the recovery times in the scenarios to 120 seconds to start? The etcd pod scenario I ran took 97 seconds:         
Also, should we document somewhere what each of the times below are? Is readiness the pod is created and then thats the time it takes to become ready? 
```
"total_recovery_time": 107.39545583724976,
"pod_readiness_time": 9.480031967163086,
"pod_rescheduling_time": 97.91542387008667
```